.github/workflows/create-tag.yml

```diff
@@ -56,14 +56,28 @@ on:
         description: 'URL for the production github-token-minter service.'
         type: 'string'
         required: true
-    outputs:
-      result:
-        description: 'Create tag result.'
-        value: '${{ jobs.create-tag.outputs.result }}'
+
+env:
+  TAG: '${{ inputs.tag }}'
+  BRANCH: '${{ inputs.branch || github.event.repository.default_branch }}'
+  ANNOTATED_TAG: '${{ inputs.annotated_tag }}'
+  MESSAGE: '${{ inputs.message || inputs.tag }}'
 
 jobs:
+  print-inputs:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: 'job summary'
+        run: |
+          echo "### Inputs" >> $GITHUB_STEP_SUMMARY
+          echo "- tag: ${{ env.TAG }}" >> $GITHUB_STEP_SUMMARY
+          echo "- branch: ${{ env.BRANCH }}" >> $GITHUB_STEP_SUMMARY
+          echo "- annotated tag: ${{ env.ANNOTATED_TAG }}" >> $GITHUB_STEP_SUMMARY
+          echo "- message: ${{ env.MESSAGE }}" >> $GITHUB_STEP_SUMMARY
```

Comment:
I think theres an injection vulnerability here. In general, dont trust anything from the user. Instead you can just reference these as environment variables instead of interpolating them from the github syntax.


echo "${TAG}" >> $GITHUB_STEP_SUMMARY

Reply:
I see, thanks for catching it!